src/configuration/data-purge-config.ts

```diff
@@ -4,7 +4,7 @@ export const dataPurgeConfigurationType = Type.Object({
   skipCommentsWhileAssigned: Type.Union([Type.Literal("all"), Type.Literal("exact"), Type.Literal("none")], {
     default: "all",
     description:
-      "Do not offer credit to comments posted by a user between the time they were first assigned and last unassigned from the task.",
+      "Determines the comments posted by a user to include in the final rewards. 'all': excludes comments between first and last assignment. 'exact': excludes comments during the exact assignment periods. 'none': includes every comment.",
```

Comment:
description:
  "Configures how user comments are included in the rewards calculation when they are assigned to a GitHub issue:
  - 'all': Excludes all comments made between the first assignment start and the last assignment end, discouraging gaming by unassigning and commenting for rewards.
  - 'exact': Excludes only comments made during precise assignment periods, targeting times when the user is actively assigned.
  - 'none': Includes all comments, regardless of assignment status or timing."
  
  ---
  
  chatgpt version

Reply:
Cannot really do the bullet point formatting since it's only a string but will update the content